src/house.h

```diff
@@ -33,23 +33,23 @@ static const HouseID NUM_HOUSES_PER_GRF = NUM_HOUSES; ///< Number of supported h
 static const uint HOUSE_NUM_ACCEPTS = 16; ///< Max number of cargoes accepted by a tile
 static const uint HOUSE_ORIGINAL_NUM_ACCEPTS = 3; ///< Original number of accepted cargo types.
 
-enum BuildingFlags : uint8_t {
-	TILE_NO_FLAG         =       0,
-	TILE_SIZE_1x1        = 1U << 0,
-	TILE_NOT_SLOPED      = 1U << 1,
-	TILE_SIZE_2x1        = 1U << 2,
-	TILE_SIZE_1x2        = 1U << 3,
-	TILE_SIZE_2x2        = 1U << 4,
-	BUILDING_IS_ANIMATED = 1U << 5,
-	BUILDING_IS_CHURCH   = 1U << 6,
-	BUILDING_IS_STADIUM  = 1U << 7,
-	BUILDING_HAS_1_TILE  = TILE_SIZE_1x1 | TILE_SIZE_2x1 | TILE_SIZE_1x2 | TILE_SIZE_2x2,
-	BUILDING_HAS_2_TILES = TILE_SIZE_2x1 | TILE_SIZE_1x2 | TILE_SIZE_2x2,
-	BUILDING_2_TILES_X   = TILE_SIZE_2x1 | TILE_SIZE_2x2,
-	BUILDING_2_TILES_Y   = TILE_SIZE_1x2 | TILE_SIZE_2x2,
-	BUILDING_HAS_4_TILES = TILE_SIZE_2x2,
+enum class BuildingFlag : uint8_t {
+	Size1x1    = 0,
```

Comment:
Are all explicit (contiguous) numbers necessary?

Reply:
Yes, these values are used as-is in the NewGRF spec.

I haven't been explicitly saying so, but I've been adding or removing explicit numbers as appropriate (if they are used in savegames, NewGRF or script.)